src/main.rs

```diff
@@ -47,9 +47,7 @@ async fn main() -> eyre::Result<()> {
             command_line::indexing::delete_index::action(matches)
         }
         (command_line::indexing::index_sdf::NAME, matches) => {
-            let writes = command_line::indexing::index_sdf::action(matches)?;
-            log::info!("wrote: {}", writes);
```

Comment:
This is an example of a log statement that is slightly better than a `println!()`